packages/arb-token-bridge-ui/tests/e2e/specs/login.cy.js

```diff
@@ -0,0 +1,16 @@
+describe("Login Account", () => {
+  before(() => {
+    cy.disconnectMetamaskWalletFromAllDapps();
+  });
+  
+  it("Should show connect wallet if not logged in", () => {
+    cy.visit(`/`);
+    cy.findByText("Agree to terms").should("be.visible").click();
+    cy.findByText("MetaMask").should("be.visible");
+    cy.findByText("Connect to your MetaMask Wallet").should("be.visible");
+  });
+  
+  it("Should connect wallet using MetaMask successfully", () => {
+    cy.login();
```

Comment:
We should maybe add an assertion below that the user is seeing `Enter amount` or some other text after successful connection.

Reply:
I added a test for "Bridging summary will appear here."
Let me know if you would like to cover more text / buttons